frontend/src/components/LoginHeader.tsx

```diff
@@ -4,6 +4,7 @@ import "../index.css";
 export const LoginHeader = () => {
   return (
     <section className="header-wrapper">
+      <img src="mstile-144x144.png" className="login-img" alt="Urdr icon" />
```

Comment:
Checkout the picture element instead: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture

The png image should be cropped to a smaller size for the larger screens and a smaller for the views less than 700, e.g (mobile first approach):
```
<picture>
  <source srcset="mstile-88x88.png" media="(min-width: 700px)" />
  <img src="mstile-64x64.png" alt="Urdr icon" />
</picture>
```
Then the separate media query is not needed anymore.

Reply:
I just choose the size and crop the image to the desired size.